src/assets/FormulaIcon.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+const FormulaIcon = () => {
```

Comment:
very minor - should we do `export default (<svg...` like in the CalculationIcon file, for consistency?